tests/unit-tests/00-check/check-eqvp.scm

```diff
@@ -12,7 +12,7 @@
 
 ;; Equivalent NaNs should be eqv?
 (let ((x (fl/ 0. 0.)))
-  (eqv? x x))
+  (check-eqv? x x))
```

Comment:
In principle yes. But that would be unexpected. But even ```(fl/ 0.0 0.0)``` and a different ```(fl/ 0.0 0.0)``` could give different NaNs... some implementations store in some of the bits of the NaN the address of the faulty operation.  So maybe this is "safe":

```
(define (f x y) (fl/ x y))
(let ((x (f 0.0 0.0)) (y (f 0.0 0.0))) (check-eqv? x y))
```